doc/classes/CollisionPolygon3D.xml

```diff
@@ -21,6 +21,9 @@
 			Array of vertices which define the polygon.
 			[b]Note:[/b] The returned value is a copy of the original. Methods which mutate the size or properties of the return value will not impact the original polygon. To change properties of the polygon, assign it to a temporary variable and make changes before reassigning the [code]polygon[/code] member.
 		</member>
+		<member name="margin" type="float" setter="set_margin" getter="get_margin" default="0.04">
+			The collision margin for the shape.
+		</member>
```

Comment:
The properties are listed alphabetically so this should go before `polygon`.

Reply:
As for the documentation, I see you've copied it from `Shape3D` so it's OK, though maybe it should say "The collision margin for the polygon."?

I feel like both this one and `Shape3D`'s `margin` descriptions could have more details on what this impacts, as it's a fairly important setting and the current description hardly does more than putting the property name in a full sentence. That's only tangentially related to this PR though, but I mention it in case you or reviewers have suggestions for better docs :)

Reply:
Agreed. For CollisionPolygon3D I would suggest:

```
The collision margin for the generated [Shape3D]. See [member Shape3D.margin] for more details.
```

And the Shape3D description can be improved indeed:

```
The collision margin for the shape. Used in Bullet Physics only.
Collision margins allows collision detection to be more efficient by adding an extra shell around shapes. Collision algorithms are more expensive when objects overlap by more than their margin, so a higher value for margins is better for performance, at the cost of accuracy around edges as it makes them less sharp.
```